pkg/codegen/nodejs/gen.go

```diff
@@ -499,38 +499,14 @@ func provideDefaultsFuncNameFromName(typeName string) string {
 // `type` is the type which the function applies to.
 // `input` indicates whither `type` is an input type.
 func (mod *modContext) provideDefaultsFuncName(typ schema.Type, input bool) string {
-	if !isProvideDefaultsFuncRequired(typ) {
+	if !codegen.IsProvideDefaultsFuncRequired(typ) {
```

Comment:
This is a no-op change correct? just moving it out to reuse?

Reply:
Yep. I just refactored so the code could be shared between Go and TypeScript codegen.